src/js/menu/menu.js

```diff
@@ -112,7 +113,7 @@ class Menu extends Component {
 
     // Close menu popup when a user clicks outside the menu
     if (!this.children().some((element) => {
-      return element.el() === relatedTarget;
+      return element.el && (element.el() === relatedTarget);
```

Comment:
This is actually a bugfix for a previous change, not specifically related to this PR; it shows up with the Chapters menu where the first child element isn't a video.js component, it's just an HTML `<li>` element which doesn't have a `.el()` method.